app/src/main/kotlin/net/corda/cli/application/utils/Files.kt

```diff
@@ -7,10 +7,13 @@ class Files {
     companion object {
 
         fun cliHomeDir(): File {
-            return if (System.getenv("CORDA_CLI_HOME_DIR").isNullOrEmpty()) {
-                Paths.get(System.getProperty("user.home"), "/.corda/cli/").toFile()
-            } else {
+
+            return if (System.getenv("CORDA_CLI_HOME_DIR")?.isNotEmpty() == true) {
```

Comment:
Out of curiosity, why did we need to change that?

Reply:
To give you more context, I had to make this change to make the failed tests work with Java version upgrade to 17. The reason why the tests that use system lambda library failed was because:
1. the library is not usable anymore in the way that we were using in tests - setting env variables
2. therefore, we can't really test these behaviours depending on env variables

I hope to seek some ideas/opinions if this change is sensible, or if any other way around and if we really need these tests if the library to set env variable no longer work?

Reply:
I found an alternative to system lambda that is not compatible with JVM 16 onward. [Here is the link](https://github.com/webcompere/system-stubs). This is basically a fork of System Lambda. This appears to be working without having to change the existing testing logic.